src/payment/payment-strategy-type.ts

```diff
@@ -19,6 +19,7 @@ enum PaymentStrategyType {
     LEGACY = 'legacy',
     OFFLINE = 'offline',
     OFFSITE = 'offsite',
+    APPROVED = 'approved',
```

Comment:
There are no statuses in this type, I think `APPROVED = 'approved',` status should be in the paypalcommerce strategy

Reply:
fixed